libsplinter/src/protos/mod.rs

```diff
@@ -81,20 +81,28 @@ where
     }
 }
 
-pub trait IntoNative<T>: Sized
+pub trait IntoNative<T>: Sized {
```

Comment:
Typo in the commit message "implemetnations"

Reply:
Updated